components/brave_new_tab_ui/components/default/gridSites/index.ts

```diff
@@ -108,26 +140,26 @@ export const TileActionsContainer = styled('nav')<{}>`
   display: flex;
 `
 
-export const TileMenu = styled('div')<{}>`
+export const TileMenu = styled('div') <{}>`
   position: absolute;
-  top: 15px;
-  left: 80px;
+  margin-left: -12px;
```

Comment:
I've added the negative margins here so that the position logic for the menu is simply "Line up the corners", and this handles the overlap.

Reply:
Maybe we can add a comment about where the exact value of `-12px` comes from or how it's calculated?

Reply:
:+1: